### DISCUSSION TITLE
Overlays Meeting (2022-03-29)

### DISCUSSION THREAD

**Discussion by webron** [OAI/Overlay-Specification#22] [Meeting Notes]
Date: 2022-03-29

## Attendees

- Josh Ponelat
- Ron Ratovsky
- Stu Waldron
- Darrel Miller

(this summary was written after the meeting and not during, so some inaccuracies can appear)

## Summary
- We've talked over JSONPath vs JMESPath and potential alternatives. 
- There are two main perspectives on how to proceed:
  - Make sure the we cover the main use cases.
  - Check how well the current specification draft covers known use cases.
- Darrel brought up the benefits of having code implementations that can help test the use cases, and find faults in the current draft.

## Action items
- Create examples for known use cases (this is open for _anyone_ who wants to help):
  - Check the current [list of use cases](https://github.com/OAI/Overlay-Specification/discussions/9#discussioncomment-1431389).
  - Pick one or more use cases you'd like to cover. Add a comment on that discussion saying which use cases you're taking on, so others won't work on the same use cases.
  - Write down examples of overlays to show case how those use cases can (or cannot) be supported based on the current draft.
  - Share those examples in the same discussion before our next meeting.
- We'll see if we can get the word out with the help of the marketing team and call for any tool builders to build implementations for overlays to help us test it out. This can be done by either large organization or OpenAPI enthusiasts who want to help us check this solution.

## Meeting recording
https://zoom.us/rec/share/8bma1xQei4e5eVxx7aDjMcOQxzshLTU_ajm5rSqcm8u-50q2RAB9qzcz8CNVFsrT.WhdCwcnEuuoaZNW9

**Comment by lornajane** [comment]
Closing this discussion from a past meeting, but it will still be available for reference.
